src/email_service/validate_input/lambda_function.py

```diff
@@ -250,8 +252,27 @@ def prepare_run_data(
     attachment_files: list[dict[str, Any]],
     current_user_info: dict[str, Any],
 ) -> dict[str, Any]:
-    """Prepare run item data."""
+    """Prepare run item data and backup files."""
     created_at = get_current_utc_time()
+    run_id = common_data["run_id"]
+
+    template_backup_key = s3_util.copy_file_to_run_folder(
+        template_info["s3_object_key"], run_id, "template"
+    )
+    template_info["backup_s3_key"] = template_backup_key
+
+    if spreadsheet_info and recipient_source == "SPREADSHEET":
+        spreadsheet_backup_key = s3_util.copy_file_to_run_folder(
+            spreadsheet_info["s3_object_key"], run_id, "spreadsheet"
+        )
+        spreadsheet_info["backup_s3_key"] = spreadsheet_backup_key
+
+    for attachment in attachment_files:
+        attachment_backup_key = s3_util.copy_file_to_run_folder(
+            attachment["s3_object_key"], run_id, "attachments"
+        )
+        attachment["backup_s3_key"] = attachment_backup_key
```

Comment:
Could you rename `backup_s3_key` to `backup_s3_object_key` for consistency with AWS documentation?
According to AWS documentation, the unique identifier for an object in a bucket is referred to as the "object key."


> The object key (or key name) uniquely identifies the object in an Amazon S3 bucket.
> Reference: https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-keys.html#:~:text=The%20object%20key%20(or%20key%20name)%20uniquely%20identifies%20the%20object%20in%20an%20Amazon%20S3%20bucket.